### DISCUSSION TITLE
Project Export Driver Version

### DISCUSSION THREAD

**Discussion by simonB2020** [dbeaver/dbeaver#22162] [General]
When exporting/importing a DBeaver project, the connections' driver is included in the "data-sources.json". 
However there is no version specified - so on import, the latest version of the driver is downloaded. 
I was hoping that we could explicitly specify a particular version.

Can't find any documentation on what attribute to add, or even it it would work. 
Blindly tried adding "version" but it just fell flat on it's face. Has anyone tried this or know of other useful info ?

Thanks

**Comment by E1izabeth** [comment]
I believe you can change the driver version in this dialog
![image](https://github.com/dbeaver/dbeaver/assets/28875055/24221327-b5e6-4210-a81d-2d6cf3c0ce31)

**Reply by simonB2020** [reply]
Thanks @E1izabeth, 
I was looking for a way to embed this setting in the Project export, to avoid the end users having to go through this step.

We have managed to do it like this : 

```
"connections": {
		"postgres-redshift-jdbc-18c40a1a63e-11fd252a6d9dc233": {
			"provider": "postgresql",
			"driver": "postgres-redshift-jdbc",
			"version": "2.1.0.21",
```
Once imported, DBeaver does download the specified version when a user first opens a connection.
Though still some unresolved questions....

When exporting a project there is a checkbox for 'include driver files' ... yet I am unsure what this does as no driver files are exported !?
